oak_functions/loader/src/server.rs

```diff
@@ -699,7 +635,8 @@ impl WasmHandler {
 
         wasm_state.invoke();
 
-        for (_, extension) in wasm_state.extensions_indices.iter_mut() {
+        // TODO(mschett): Make more idiomatic.
```

Comment:
Here I am not sure how I can deal with the `?` if I use something like `.for_each()` to iterate. Is there a better way?

Reply:
maybe something like this? (I haven't actually tried it)

```rust
wasm_state.extensions.iter_mut().map(|e| e.terminate()).collect::<Result<Vec<_>, _>>()?;
```

Reply:
That worked! Thank you!